src/components/Filter/FilterContent/FilterContent.tsx

```diff
@@ -236,7 +236,7 @@ const FilterContent: React.FC<FilterContentProps> = ({
                 )}
                 {filter.multipleFields ? (
                   <CollectionWithDividers
-                    collection={filter.multipleFields}
+                    collection={filter?.multipleFields}
```

Comment:
This check won't make sense because you're not doing optional chain above, if filter is undefined it would not get to this code because it would fail at line 237 at:
```
{filter.multipleFields ? )
```